### ISSUE TITLE
Hundreds of errors appearing in console when on raw.github.com

### ISSUE THREAD

**Issue by callumlocke** [facebook/react-devtools#1022]
When I view anything on `raw.github.com`, such as the [readme for this repo](https://raw.githubusercontent.com/facebook/react-devtools/master/README.md), the Chrome react-devtools extension causes an error to appear in the console 10–20 times per second:

```
Blocked script execution in '<URL>' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.
```

![image](https://user-images.githubusercontent.com/250617/39433410-b9bd40c0-4c8d-11e8-80f3-793c0b066e84.png)

I don't know if it happens on any other sites.

**Comment by bvaughn** [comment]
The only place we reference `README.md` in this extension source is in [the disabled popup HTML](https://github.com/facebook/react-devtools/blob/ed27c461ad4057fe08c30854383bbacc06f9708b/shells/webextension/popups/disabled.html#L20), and this is just a simple link.

However, I can confirm that I see this Chrome console error as well, and disabling the React DevTools extension "fixes" it.

That being said, I see the blocked log only once per page load- not hundreds of times as you've reported.

I'm not sure what's going on here.

**Comment by bvaughn** [comment]
React DevTools has been rewritten and [recently launched a new version 4 UI](https://reactjs.org/blog/2019/08/15/new-react-devtools.html). The source code for this rewrite was done in a separate repository and now lives in the main React repo ([github.com/facebook/react](https://github.com/facebook/react/)).

Because version 4 was a total rewrite, and all issues in this repository are related to the old version 3 of the extension, I am closing all issues in this repository. If you can still reproduce this issue, or believe this feature request is still relevant, please open a new issue in the React repo: https://github.com/facebook/react/issues/new?labels=Component:%20Developer%20Tools
